gc.rb

```diff
@@ -30,12 +30,12 @@ module GC
   #  Note: These keyword arguments are implementation and version dependent. They
   #  are not guaranteed to be future-compatible, and may be ignored if the
   #  underlying implementation does not support them.
-  def self.start full_mark: true, immediate_mark: true, immediate_sweep: true
-    __builtin_gc_start_internal full_mark, immediate_mark, immediate_sweep
+  def self.start full_mark: true, immediate_mark: true, immediate_sweep: true, compact: false
+    __builtin_gc_start_internal full_mark, immediate_mark, immediate_sweep, compact
   end
 
   def garbage_collect full_mark: true, immediate_mark: true, immediate_sweep: true
-    __builtin_gc_start_internal full_mark, immediate_mark, immediate_sweep
+    __builtin_gc_start_internal full_mark, immediate_mark, immediate_sweep, false
```

Comment:
don't you need compact option?

Reply:
No, I think maybe we should only have `GC.compact`.  I want to switch `GC.compact` to use `__builtin_gc_start_internal`, but I need to fix heap sorting.